### PR TITLE
More IRU coat options.

### PR DESCRIPTION
**Pull request by Aboshedab**
More loadout options for the IRU coats.

Instead of the single black IRU coat, you now have 3 styles, each in black or brown options.

### CODE REVIEW

**Comment by Aboshedab** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm [hunk 1]
```diff
@@ -87,8 +87,19 @@
 	sort_category = "Xenowear - IPC"
 
 /datum/gear/suit/idris
-	display_name = "Idris Reclamation Unit coat"
+	display_name = "Idris Reclamation Unit coats"
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm]:
```suggestion
	display_name = "Idris Reclamation Unit coat selection"
```

**Comment by Aboshedab** [comment]
Not merging this due to potential changes.

**Comment by Aboshedab** [comment]
Potential changes added.

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -564,15 +564,45 @@
 	pockets.max_storage_space = 8
 
 /obj/item/clothing/suit/armor/vest/idris
-	name = "Idris Reclamation Unit coat"
-	desc = "A coat worn by the Idris reclamation units, notorious across space."
+	name = "Black Idris Unit coat"
```
Comment (Alberyk) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	name = "black Idris Unit coat"
```

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -564,15 +564,45 @@
...
+	desc = "A coat worn by the Idris units, notorious across space."
 	icon_state = "iru_coat"
 	item_state = "iru_coat"
 	cold_protection = 0
 	min_cold_protection_temperature = 0
 	heat_protection = 0
 	max_heat_protection_temperature = 0
 
+/obj/item/clothing/suit/armor/vest/idris/brown
+	name = "Brown Idris Unit coat"
```
Comment (Alberyk) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	name = "brown Idris Unit coat"
```

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -564,15 +564,45 @@
...
+	desc = "A coat worn by the Idris units, notorious across space. This one is brown."
+	icon_state = "iru_coat_brown"
+	item_state = "iru_coat_brown"
+
+/obj/item/clothing/suit/armor/vest/idris/trenchcoat/black
+	name = "Black Idris Unit trench coat"
```
Comment (Alberyk) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	name = "black Idris Unit trench coat"
```

File: code/modules/clothing/suits/armor.dm [hunk 5]
```diff
@@ -564,15 +564,45 @@
...
+	desc = "A trench coat worn by the Idris units, notorious across space. This one is black."
+	icon_state = "iru_trench_black"
+	item_state = "iru_trench_black"
+
+/obj/item/clothing/suit/armor/vest/idris/trenchcoat/brown
+	name = "Brown Idris Unit trench coat"
```
Comment (Alberyk) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	name = "brown Idris Unit trench coat"
```

File: code/modules/clothing/suits/armor.dm [hunk 6]
```diff
@@ -564,15 +564,45 @@
...
+	desc = "A trench coat worn by the Idris units, notorious across space. This one is brown."
+	icon_state = "iru_trench_brown"
+	item_state = "iru_trench_brown"
+
+/obj/item/clothing/suit/armor/vest/idris/duster/black
+	name = "Black Idris Unit duster coat"
```
Comment (Alberyk) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	name = "black Idris Unit duster coat"
```

File: code/modules/clothing/suits/armor.dm [hunk 7]
```diff
@@ -564,15 +564,45 @@
...
+	desc = "A duster coat worn by the Idris units, notorious across space. This one is black."
+	icon_state = "iru_duster_black"
+	item_state = "iru_duster_black"
+
+/obj/item/clothing/suit/armor/vest/idris/duster/brown
+	name = "Brown Idris Unit duster coat"
```
Comment (Alberyk) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	name = "brown Idris Unit duster coat"
```

**Comment by Aboshedab** [comment]
I broke everything.

**Comment by Aboshedab** [comment]
I fixed everything....

**Comment by Aboshedab** [comment]
I ran out of children to sacrifice. Wezzy will do this.

**Comment by Aboshedab** [comment]
Fixed everything, I promise.

Thanks to a bunch of folks for helping me.

**Pull request merged**